Remove default connection details from CLI (#396)
comit-network/xmr-btc-swap: CHANGELOG.md

- A changelog file.

### Changed

> - Require swap-cli user to specify the connection details of the peer they wish to swap with.

Reviewer: ```suggestion
- Require the `swap` CLI user to specify the connection details of the peer they wish to swap with.
```

Author: what about buyer instead of swap-cli user?

>   The swap-cli no longer defaults to connecting to COMIT if connection details are not provided.
>   Connecting buyers to us by default is not consistent with our vision of a decentralised network of sellers.

Reviewer: This "connecting to COMIT" is a bit vague given that there exists this idea of the COMIT network :D

What do you think of something along the lines of:

```suggestion
  Throughout the public demo phase of this project, the CLI defaulted to an `asb` with peer-ID `12D3KooWCdMKjesXMJz1SiZ7HgotrxuqhQJbP5sgBm2BwP1cqThi` and multi-address `/dns4/xmr-btc-asb.coblox.tech/tcp/9876`.
  Having the defaults made it easy for us to give something to the community that can easily be tested, however it is not aligned with our long-term vision of a decentralised network of sellers.
```

Author: I avoided asb because buyers may not know/care what that is. How about:

Throughout the public demo phase of this project, the CLI traded with us by default using with peer-ID `12D3KooWCdMKjesXMJz1SiZ7HgotrxuqhQJbP5sgBm2BwP1cqThi` and multi-address `/dns4/xmr-btc-asb.coblox.tech/tcp/9876`.
  Having the defaults made it easy for us to give something to the community that can easily be tested, however it is not aligned with our long-term vision of a decentralised network of sellers.

Reviewer: Works for me :)

- A changelog file.

### Changed

- Require the buyer to specify the connection details of the peer they wish to swap with.

>   Throughout the public demo phase of this project, the CLI traded with us by default if the peer id and multiaddress of the seller was not specified.

Reviewer: ```suggestion
  Throughout the public demo phase of this project, the CLI traded with us by default if the peer id and multiaddress of the seller were not specified.
```

- Automatic resume of unfinished swaps for the `asb` upon startup.
  Unfinished swaps from earlier versions will be skipped.

### Changed

- Require the buyer to specify the connection details of the peer they wish to swap with.
  Throughout the public demo phase of this project, the CLI traded with us by default if the peer id and multiaddress of the seller were not specified.
  Having the defaults made it easy for us to give something to the community that can easily be tested, however it is not aligned with our long-term vision of a decentralised network of

>   sellers.

Reviewer: Still got a stray newline here :)

>   sellers.

Reviewer: Stick to one sentence per line please. Adjust dprint settings if necessary.
